sdk/core/abort-controller/review/abort-controller.api.md

```diff
@@ -13,6 +13,7 @@ export class AbortError extends Error {
 export interface AbortSignalLike {
     readonly aborted: boolean;
     addEventListener(type: "abort", listener: (this: AbortSignalLike, ev: any) => any, options?: any): void;
+    readonly reason?: any;
```

Comment:
see https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/node/globals.d.ts#L201

and MDN: https://developer.mozilla.org/en-US/docs/Web/API/AbortSignal/reason#value